src/service.ts

```diff
@@ -60,4 +60,22 @@ export default class ElectronWorkerService implements Services.ServiceInstance {
       }
     }
   }
+
+  async afterCommand(commandName: string) {
+    if (commandName === 'execute') {
```

Comment:
Why do we run this only after `execute` commands?

Reply:
Also I wonder how feasible it is to check if any mocks have been registered before and skip this if there aren't any.

Reply:
Perhaps we can store a value of the number of mocks registered and remove the `execute` check.  This PR is just a rough prototype atm, the mock interface needs updating for the new approach amongst other things.

Reply:
Sounds good. I would just put the mock store into a file and have it exported only a single instance. This way you can import it from everywhere to access its values, kind a like a global